tests/unit/moduleapi/defrag.tcl

```diff
@@ -46,7 +46,8 @@ start_server {tags {"modules"} overrides {{save ""}}} {
 
             after 2000
             set info [r info defragtest_stats]
-            assert {[getInfoProperty $info defragtest_global_attempts] > 0}
+            assert {[getInfoProperty $info defragtest_global_strings_attempts] > 0}
+            assert {[getInfoProperty $info defragtest_global_dicts_attempts] > 0}
```

Comment:
maybe we can change the test to be a lot bigger so that it really breaks in the middle of a dict defrag?

Reply:
Add a new info `defragtest_global_dicts_pauses` to check the breaks in the mid.